src/Assets/NuGet.Config

```diff
@@ -0,0 +1,5 @@
+<configuration>
+  <packageSources>
+    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/packages/index.json" />
```

Comment:
My only concern is that this won't break prod-con. @johnbeisner or @tannergooding please speak up to it.

Reply:
I chatted with Tanner about this and it sounds like this is the feed that will be used for prod-con.  There is some capability to inject additional feeds in the build infrastructure, but it's not being used and wouldn't have flowed through to the tests before this PR either.